libkineto/src/output_csv.cpp

```diff
@@ -56,14 +57,17 @@ void EventCSVLogger::handleSample(
       if (eventNames_.find(s.name) == eventNames_.end()) {
         continue;
       }
-      *out_ << fmt::format("{:%Y-%m-%d %H:%M:%S}", tm_result) << ",";
-      *out_ << sample.deltaMsec << ",";
-      *out_ << device << ",";
-      *out_ << s.name;
+      fmt::print(
+          *out_,
+          "{:%Y-%m-%d %H:%M:%S},{},{},{},",
+          tm_result,
+          sample.deltaMsec,
+          device,
+          s.name);
       for (const auto& p : s.percentileValues) {
```

Comment:
So close to being a good fmt::join, will be easy when we get C++20 range transforms